compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUMaterializeEncodingPass.cpp

```diff
@@ -4,6 +4,8 @@
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+#include <iree/compiler/Codegen/Dialect/IREECodegenDialect.h>
```

Comment:
Nit: I think the convention is to use `"..."` .. (I think there is a subtle difference between the two).